dev-lang/fennel/fennel-1.2.1.recipe

```diff
@@ -0,0 +1,38 @@
+SUMMARY="Lisp-like language that compiles to Lua"
+DESCRIPTION="
+Fennel is a programming language that brings together the speed, simplicity, \
```

Comment:
Keep DESCRIPTION start on one line:
```diff
-DESCRIPTION="
-Fennel is a programming language that brings together the speed, simplicity, \
+DESCRIPTION="Fennel is a programming language that brings together the speed, simplicity, \
```

Reply:
Fixed in 1753cdda8ef547455177509651a671bc28a3e5bd.